admin/backend/auth.py

```
@@ -0,0 +1,185 @@
+"""
+Authentication utilities for the admin dashboard.
+"""
+
+import hashlib
```

Ruff (0.12.2)

`hashlib` imported but unused

Remove unused import: `hashlib`

(F401)